include/dlaf/eigensolver/bt_band_to_tridiag/impl.h

```diff
@@ -259,13 +259,95 @@ struct TileAccessHelper {
   Part part_top_;
   Part part_bottom_;
 };
+
+template <Backend B, Device D, class T>
+struct Helper;
+
+template <class T>
+struct Helper<Backend::MC, Device::CPU, T> {
```

Comment:
I guess this still needs a renaming. Is it also a `ComputePanelHelper` like in https://github.com/eth-cscs/DLA-Future/pull/493?

Reply:
I fully agree, we have to find a better name before merging.

This helper, given an input tile with "compact" form reflectors, extracts and return them well-formed together with the T factor useful for their application in block.

Any name suggestion? @msimberg @rasolca @teonnik

Reply:
Some random suggestions:
`HHManager`, `HHMatricesHelper`
You can also replace HH with HHR if you find it easier to understand...